src/components/ui/button.tsx

```
@@ -0,0 +1,58 @@
+  ...props
+}: ButtonPrimitive.Props & VariantProps<typeof buttonVariants>) {
+  return (
+    <ButtonPrimitive
+      data-slot="button"
+      className={cn(buttonVariants({ variant, size, className }))}
+      {...props}
+    />
+  )
+}
+
+export { Button, buttonVariants }
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)